server.js

```diff
@@ -1,11 +1,135 @@
 const express = require('express')
 const app = express()
+const cors = require('cors')
+const bodyParser = require('body-parser')
+const { Dvr } = require('@mui/icons-material')
 const port = 5000
 
+const sqlite3 = require('sqlite3').verbose();
+
+const db = new sqlite3.Database('./shopListDB.db', sqlite3.OPEN_READWRITE, (err) => {
+  if(err) 
+    return console.error(err.message);
+
+  console.log('connection with database successful');
+});
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use(cors({
+  origin: '*',
+  credentials: true
+}));
+
 app.get('/', (req, res) => {
```

Comment:
Remove

Reply:
Need it to run server and client on same localhost

Reply:
I meant the hello world